### PR TITLE
Fix instance state filter menu on firefox browser

### PR DESCRIPTION
**Pull request by paulino-corujo**
@manolakis

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/ek_pull_request/140/ for more details.

**Comment by arnaud-elasticbox** [comment]
Jenkins, test this please.

**Comment by allyourbotsarebelongtous** [comment]
Deployed test instance is available at http://ek-102.internal.elastickube.com

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/ek_pull_request/143/ for more details.

File: src/ui/app/instances/ek-instance-filters/ek-instance-filters.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <ek-instance-state-selector instances="ctrl.instancesToFilter"
-                            selected-state="ctrl.selectedState"></ek-instance-state-selector>
+                            selected-state="ctrl.selectedState"
+                            layout="column" layout-align="start start"></ek-instance-state-selector>
```
Comment (manolakis) [inline src/ui/app/instances/ek-instance-filters/ek-instance-filters.html]: please, set the internal layout of the component inside the directive.

**Comment by manolakis** [comment]
ACK

**Comment by allyourbotsarebelongtous** [comment]
Deployed test instance is available at http://ek-102.internal.elastickube.com

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/ek_pull_request/145/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://elasticbox.com/services/instances/i-j3fcoi

**Pull request merged**